frontend/cypress/tests/chartView/chartView_zoomFeature.js

```diff
@@ -0,0 +1,47 @@
+describe('zoom features in code view', () => {
+  const zoomKey = Cypress.platform === 'darwin' ? '{meta}' : '{ctrl}';
+  it('click on view commits button', () => {
+    Cypress.wait();
+
+    cy.get('.icon-button.fa-list-ul')
+        .should('be.visible')
+        .first()
+        .click();
+
+    cy.get('#tab-zoom')
+        .should('be.visible');
+  });
+
+  it('zoom into ramp', () => {
+    Cypress.wait();
+
+    // clicking from the 10th px to the 50th px in the ramp
+    cy.get('body').type(zoomKey, { release: false })
+        .get('#summary-charts .summary-chart__ramp .ramp')
+        .first()
+        .click(10, 20)
+        .click(50, 20);
+
+    cy.get('#tab-zoom')
+        .should('be.visible');
+  });
+
+  it('zoom into ramp when merge group', () => {
+    Cypress.wait();
+
+    cy.get('#summary label.merge-group > input:visible')
+        .should('be.visible')
+        .check()
+        .should('be.checked');
+
+    // clicking from the 10th px to the 50th px in the ramp
+    cy.get('body').type(zoomKey, { release: false })
+        .get('#summary-charts .summary-chart__ramp .ramp')
+        .first()
+        .click(10, 20)
+        .click(50, 20);
+
+    cy.get('#tab-zoom')
```

Comment:
Maybe can test for reload here as well to see if the zoom selection is preserved?

Reply:
I actually intend to leave that as a separate PR that tests all things related to zoom. Currently, this PR only tests all things related to summary

Reply:
or do you mean the dashed lines when activating zoom? those lines are not preserved after reloading currently.

Reply:
I was talking about the zoom tab on the right. Wasn't sure if that was considered part of this PR. Guess not.